docs/src/user-guide/quick-start/overview.md

```diff
@@ -0,0 +1,151 @@
+# Overview
+
+This page will guide you through CLP's system requirements and help you choose which flavour of CLP will be best for you.
+
+If you already have CLP set up, click your flavour of CLP below to learn how to use it.
+
+::::{grid} 1 1 2 2
+:gutter: 2
+
+:::{grid-item-card}
+:link: clp-json
+CLP for JSON logs
+^^^
+Learn about compressing and searching JSON logs.
+:::
+
+:::{grid-item-card}
+:link: clp-text
+CLP for unstructured text logs
+^^^^^^^^^^^^
+Learn about compressing and searching unstructured text logs.
+:::
+::::
+
+---
+
+## System Requirements
+
+To run properly on your system, CLP requires a few other programs. Before you set up CLP, ensure that you have the following installed.
+
+### Docker
+
+CLP uses Docker to deploy its different components. You can check whether Docker is already installed on your system by running the command
+
+```bash
+docker version
+```
+
+If Docker is not on your system, follow the instructions [here][Docker] to install it.
+
+If you're not running as root, ensure Docker can be run [without superuser privileges][docker-non-root].
+
+If you're using Docker Desktop, ensure version 4.34 or higher is installed, and [host networking is enabled][docker-desktop-host-networking].
+
+### Python
+
+CLP uses Python to interpret the scripts that coordinate how it runs. Specifically, CLP needs Python version 3.8 or higher. Many Linux distributions come with Python pre-installed; to confirm that it's on your system, run the command
+
+```bash
+python3 --version
+```
+
+If Python isn't on your system, or if the version isn't high enough, install or upgrade it.
+
+:::{note}
+If you're planning to deploy CLP on multiple nodes/systems, there are a few other system requirements; check out the [multi-node deployment](../guides-multi-node) page for more details.
+:::
+
```

Comment:
Organizationally, this is a bit awkward to place here (e.g., a reader might think it's part of the Python section). I think let's just delete it and the user can figure it out when they read the multi-node guide.